### PR TITLE
Workshop1

### PR DESCRIPTION
**Pull request by jithendraathipatla**
Added photos in the image folder

### CODE REVIEW

File: workshop_1.html [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+                        <div class="col-md-2">
+                            <div id="align">
+                                <h4 style="color:#ff9900">Workshop Trainer</h4>
+                                <h4><a href="https://www.linkedin.com/in/madhusudanshekar/" >Madhusudhan Shekar</a></h4>
+                                <p><a href="https://aws.amazon.com/">AWS</a></p>
+                            </div>
+                        </div>
+
+                        <div class="col-md-2">
+                            <img id="myImg" src="http://inserverless.com/images/speakers/Srushith.jpg" alt="Snow">
```
Comment (SathyaBhat) [inline workshop_1.html]: let's not source images from outside the site, upload and reference locally so it gets distributed with Cloudfront

File: workshop_1.html [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+                </div>
+
+                <div class="modal-footer">
+                    <br>
+                    <div class="row">
+                        <div class="col-md-2">
+                        </div>
+                        <div class="col-md-2">
+
+                            <img id="myImg" src="http://www.containerconf.in/images/speakers/Madhu.jpg" alt="Snow">
```
Comment (SathyaBhat) [inline workshop_1.html]: let's not source images from outside the site, upload and reference locally so it gets distributed with Cloudfront

File: workshop_2.html [hunk 3]
```diff
@@ -0,0 +1,249 @@
+<!DOCTYPE html>
```
Comment (SathyaBhat) [inline workshop_2.html]: file name as `deep-learning-workshop.html`

Comment (jithendraathipatla) [reply workshop_2.html]: created a page with deep-learning workshop

Comment (jithendraathipatla) [reply workshop_2.html]: Images are taken within the images folder, have not included any external source for images.

File: index.html [hunk 4]
```diff
@@ -666,12 +715,54 @@ <h4 class="headin-ttl">Workshop</h4>
           <div class="sponsors">
             <h4 class="main-heading-ttl text-center">Agenda</h4>
             <div class="text-center" style="font-size: 8vw; color: #ddd">
-              Coming soon
+            <p>Comming soon!</p>
```
[the remark this answers is not in the document]
Comment (jithendraathipatla) [reply index.html]: It has been replace with 'coming'

File: index.html [hunk 5]
```diff
@@ -576,7 +625,7 @@ <h4 class="heading-ttl">
           <!-- <img src="images/bgda.svg"> -->
           <div class="cfp">
             <h4 class="main-heading-ttl">Submit a talk</h4>
-            <h4>Last date for submission - Extended till April 15</h4>
+            <h4>Last date for submission - Apr 10, 2019</h4>
```
[the remark this answers is not in the document]
Comment (jithendraathipatla) [reply index.html]: It is rectified.

**Pull request closed without merging**